gradle/libs.versions.toml

```diff
@@ -0,0 +1,44 @@
+[versions]
+hypertrace-serviceFramework = "0.1.43"
+hypertrace-grpcUtils = "0.9.0"
+hypertrace-documentStore = "0.7.18"
+
+protoc = "3.21.2"
+grpc = "1.48.0"
+mockito = "4.8.1"
+jackson = "2.13.4.2"
+
+[libraries]
+hypertrace-serviceFramework-grpcFramework = { module = "org.hypertrace.core.serviceframework:platform-grpc-service-framework", version.ref = "hypertrace-serviceFramework" }
+hypertrace-serviceFramework-integrationTestFramework = { module = "org.hypertrace.core.serviceframework:integrationtest-service-framework", version.ref = "hypertrace-serviceFramework" }
+hypertrace-grpc-client-utils = { module = "org.hypertrace.core.grpcutils:grpc-client-utils", version.ref = "hypertrace-grpcUtils"}
+hypertrace-grpc-client-rx-utils = { module = "org.hypertrace.core.grpcutils:grpc-client-rx-utils", version.ref = "hypertrace-grpcUtils"}
+hypertrace-grpc-context-utils = { module = "org.hypertrace.core.grpcutils:grpc-context-utils", version.ref = "hypertrace-grpcUtils"}
+hypertrace-document-store = { module = "org.hypertrace.core.documentstore:document-store", version.ref = "hypertrace-documentStore" }
+
+grpc-netty = { module = "io.grpc:grpc-netty", version.ref = "grpc" }
+grpc-stub = { module = "io.grpc:grpc-stub", version.ref = "grpc" }
+grpc-protobuf = { module = "io.grpc:grpc-protobuf", version.ref = "grpc" }
+grpc-api = { module = "io.grpc:grpc-api", version.ref = "grpc" }
+grpc-core = { module = "io.grpc:grpc-core", version.ref = "grpc" }
+
+protobuf-java-util = { module = "com.google.protobuf:protobuf-java-util", version.ref = "protoc" }
+
+jackson-databind = { module = "com.fasterxml.jackson.core:jackson-databind", version.ref = "jackson" }
+rxjava = { module = "io.reactivex.rxjava3:rxjava", version = "3.1.3" }
+javax-annotation = { module = "javax.annotation:javax.annotation-api", version = "1.3.2" }
+google-guava = { module = "com.google.guava:guava", version = "31.1-jre" }
+findbugs-jsr = { module = "com.google.code.findbugs:jsr305", version = "3.0.2" }
+uuid-creator = { module = "com.github.f4b6a3:uuid-creator", version = "5.1.0" }
+typesafe-config = { module = "com.typesafe:config", version = "1.4.2" }
+lombok = { module = "org.projectlombok:lombok", version = "1.18.22" }
+slf4j-api = { module = "org.slf4j:slf4j-api", version = "1.7.32" }
+apache-log4j-slf4jImpl = { module = "org.apache.logging.log4j:log4j-slf4j-impl", version = "2.17.1" }
```

Comment:
could you do a quick check on version numbers. I think a few of these are outdated - log4j, rxjava and lombok at least